dscp-node/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@digicatapult/dscp-node",
+  "version": "3.0.1",
+  "description": "VITALam Node API types",
```

Comment:
VitalAM Node

Reply:
thanks, thought I'd purged all the vitalams